packages/types/src/cart/mutations.ts

```diff
@@ -0,0 +1,133 @@
+export interface UpsertAddressDTO {
+  customer_id?: string
+  company?: string
+  first_name?: string
+  last_name?: string
+  address_1?: string
+  address_2?: string
+  city?: string
+  country_code?: string
+  province?: string
+  postal_code?: string
+  phone?: string
+  metadata?: Record<string, unknown>
+}
+
+export interface UpdateAddressDTO extends UpsertAddressDTO {
+  id: string
+}
+
+export interface CreateAddressDTO extends UpsertAddressDTO {}
+
+export interface CreateCartDTO {
+  region_id?: string
+  customer_id?: string
+  sales_channel_id?: string
+
+  email?: string
+  currency_code: string
+
+  shipping_address?: CreateAddressDTO | UpdateAddressDTO
+  billing_address?: CreateAddressDTO | UpdateAddressDTO
+
+  metadata?: Record<string, unknown>
+}
+
+export interface UpdateCartDTO {
+  id: string
+  region_id?: string
+  customer_id?: string
+  sales_channel_id?: string
+
+  email?: string
+  currency_code?: string
+
+  billing_address?: CreateAddressDTO | UpdateAddressDTO
+  shipping_address?: CreateAddressDTO | UpdateAddressDTO
+
+  metadata?: Record<string, unknown>
+}
+
+export interface CreateLineItemTaxLineDTO {
+  description?: string
+  tax_rate_id?: string
+  code: string
+  rate: number
+  provider_id?: string
+}
+
+export interface CreateLineItemAdjustmentDTO {
+  code: string
+  amount: number
+  description?: string
+  promotion_id?: string
+  provider_id?: string
+}
+
+export interface UpdateLineItemTaxLineDTO {
+  id: string
+  description?: string
+  tax_rate_id?: string
+  code?: string
+  rate?: number
+  provider_id?: string
+}
+
+export interface UpdateLineItemAdjustmentDTO {
+  id: string
+  code?: string
+  amount?: number
+  description?: string
+  promotion_id?: string
+  provider_id?: string
+}
+
+export interface CreateLineItemDTO {
+  title: string
```

Comment:
```suggestion
  name: string
```

This has been changed to name right?

Reply:
Yes, nice catch.

Reply:
No wait, this was only on the shipping method. Not the line item

Reply:
ah yeah, my bad. Wrong file! :D